src/get-started/java-binding.md

```diff
@@ -362,6 +392,24 @@ service:
 - Follow the instructions in the [application guide][app-tutorial] to configure
   and start an Exonum node with your service.
 
+## Common Library
+
+Java Binding includes a library module that can be useful for Java clients and
```

Comment:
What are Java clients? Maybe Java client applications that communicate/need to communicate/interact with an Exonum service?